src/track/track_gps_l2cm.h

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (C) 2016 Swift Navigation Inc.
+ * Contact: Jacob McNamee <jacob@swiftnav.com>
+ * Contact: Adel Mamin <adelm@exafore.com>
+ *
+ * This source is subject to the license found in the file 'LICENSE' which must
+ * be be distributed together with this source. All other rights reserved.
+ *
+ * THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND,
+ * EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
+ */
+#ifndef SWIFTNAV_TRACK_GPS_L2CM_H
+#define SWIFTNAV_TRACK_GPS_L2CM_H
+
+#include <libswiftnav/common.h>
+
+#ifdef FEATURE_TRACK_GPS_L2CM
```

Comment:
I still like `TRACK_GPS_L2CM_INTERNAL` or similar so that it's clear that users should never define this value.